packages/plans-grid/src/plans-table/style.scss

```diff
@@ -228,6 +228,10 @@
 		color: var( --studio-gray-90 );
 		text-decoration: line-through;
 
+		span {
+			opacity: 0.5;
+		}
+
```

Comment:
I know this may sound a bit of a nitpick, but I think we should apply those styles to the whole `.components-button` button — thus overriding the default `opacity: 0.3`

It looks like we could just move the

```scss
&:disabled {
  opacity: 0.5;
}
```

rule from being specific for the primary button, to instead apply to all `.plan-item__select-button.components-button` buttons — what do you think?

Reply:
I agree that remaining consistency is the best option but that's where it gets confusing as well. According to the design, the border opacity should be 0.64...

<img width="664" alt="Screenshot 2020-11-30 at 13 49 27" src="https://user-images.githubusercontent.com/12104589/100612296-0cf96c00-3313-11eb-9eb8-e456704c87b6.png">

I'm all in favor of applying `opacity: 0.5;` to both border & label. What should we do?

Reply:
Let's apply `opacity: 0.5` to the whole `button` for now. We can always add a custom `border-color` later on